src/Prometheus/Collector.php

```diff
@@ -101,4 +97,24 @@ protected function assertLabelsAreDefinedCorrectly(array $labels): void
             throw new InvalidArgumentException(sprintf('Labels are not defined correctly: %s', print_r($labels, true)));
         }
     }
+
+    /**
+     * @param string $metricName
+     */
+    public static function validateMetricName(string $metricName): void
```

Comment:
I'd say change this to `assert` instead.
Validations return `bool`, this is throwing exception.

Reply:
Good point, i changed it.